### ISSUE TITLE
module DocuSign.Model.DateTime is not available

### ISSUE THREAD

**Issue by loucash** [tandemequity/docusign_elixir#10]
Hey,

I was playing with this library and I get this error when calling
` DocuSign.Api.Templates.templates_get_template(conn, account_id, template_id)`

```
** (UndefinedFunctionError) function DocuSign.Model.DateTime.__struct__/0 is undefined (module DocuSign.Model.DateTime is not available)
    DocuSign.Model.DateTime.__struct__()
    (docusign) deps/docusign/lib/docusign/deserializer.ex:20: DocuSign.Deserializer.deserialize/5
    (docusign) lib/docusign/model/envelope_recipient_tabs.ex:81: Poison.Decoder.DocuSign.Model.EnvelopeRecipientTabs.decode/2
    (elixir) lib/map.ex:752: Map.update!/3
    lib/poison/decoder.ex:24: anonymous fn/5 in Poison.Decode.transform/4
    (elixir) lib/enum.ex:1925: Enum."-reduce/3-lists^foldl/2-0-"/3
    lib/poison/decoder.ex:24: Poison.Decode.transform/4
    (elixir) lib/map.ex:752: Map.update!/3
```

**Comment by loucash** [comment]
I replaced `DocuSign.Model.DateTime` with just `DateTime` and it did a trick.

A comment in a file tells that it was generated by swagger.
How does this process look like? I couldn't find any Makefile.

**Comment by neilberkman** [comment]
@loucash I believe we did:

``` shell
java -jar swagger-codegen-cli.jar generate \
  -i https://raw.githubusercontent.com/docusign/eSign-OpenAPI-Specification/master/esignature.rest.swagger.json \
  -l elixir -o /tmp/elixir_api_client
```

**Comment by loucash** [comment]
@neilberkman thank you, this is very helpful.

**Comment by neilberkman** [comment]
@loucash fixed in 0c50ab173b8e38c2b7d0144a897fe8379be7777e
